h2/src/main/org/h2/util/MathUtils.java

```diff
@@ -28,6 +29,17 @@
      */
     static volatile boolean seeded;
 
+	/**
+	 * Android versions that less than 21 doesn't support Java 7 and class ThreadLocalRandom is not available.
+	 * For older versions will be used base Random class covered by ThreadLocal.
+	 */
+	private static ThreadLocal<Random> sRandom = new ThreadLocal<Random>() {
```

Comment:
There are three problems.

1. https://github.com/h2database/h2database/blob/master/h2/src/test/org/h2/test/unit/TestClearReferences.java
This test checks whether each field can be cleared by a Tomcat web server in some specific configurations. It detected that this field can be cleared. You should add reinitialization code and mark this field as refreshable or fix this problem in some other way.

2. I don't think that there is any reason to initialize this field when it will not be used, but this is not a big problem.

3. Formatting here is also wrong. Use spaces instead.